src/types/grafanaTypes.ts

```diff
@@ -52,6 +52,7 @@ export const DEFAULT_QUERY: Partial<BitmovinAnalyticsDataQuery> = {
   filter: [],
   resultFormat: 'time_series',
   interval: 'AUTO',
+  percentileValue: 95,
```

Comment:
this looks like fix/feature that has nothing to do with playwright ? could it be done in another PR and explained ?

Reply:
True, I just discovered it writing the tests, will create a seperate PR for this change

Reply:
Adressed in https://github.com/bitmovin/analytics-grafana-datasource/pull/95